app/models/state_file_md_intake.rb

```diff
@@ -211,6 +211,22 @@ def address
     end
   end
 
+  def filer_1099_rs(primary_or_spouse)
+    state_file1099_rs.filter do |state_file_1099_r|
+      state_file_1099_r.recipient_ssn == send(primary_or_spouse).ssn
+    end
+  end
+
+  def sum_1099_r_followup_type_for_filer(filer_1099_rs, followup_type)
```

Comment:
do we need to pass in the `filer_1099_rs` here? or instead should we pass in primary or spouse?

Reply:
ooh we can pass in the `primary_or_spouse` here and call the `filer_1099_rs` method from within this method!